app/views/pair_requests/_form.html.erb

```diff
@@ -1,23 +1,32 @@
 <%= turbo_frame_tag "new_request_form" do %>
-  <%= form_with(model: pair_request, class: 'p-4 w-full mx-auto border-2 border-primary rounded-md flex flex-col') do |form| %>
-    <div id="form-container" class='flex flex-col space-y-4 xl:space-y-0 xl:max-w-full xl:inline-flex xl:justify-around xl:space-x-8 xl:flex-row'>
-      <div class="flex justify-between space-x-4 align-middle">
-        <%= form.label :invitee_id, class: "label label-text" %>
-        <%= form.collection_select :invitee_id, User.excluding(current_user), :id, :full_name, {}, { class: 'select select-primary' } %>
+  <%= form_with(model: pair_request, class: 'p-4 mx-auto border-2 border-primary rounded-md flex flex-col') do |form| %>
+    <div id="form-container" class='flex flex-col space-y-4 xl:space-y-0 xl:max-w-full xl:inline-flex xl:justify-around xl:space-x-4 xl:flex-row object-contain' data-controller="pair-requests-form">  
+      <div class="flex justify-between space-x-4 align-middle items-center ">
+         <%= form.label :invitee_id, class: "label label-text" %>
+         <%= form.select :invitee_id, options_for_select(User.excluding(current_user).map{|invitee| [invitee.full_name, invitee.id, {"data-timezone" => invitee.time_zone}]}), {:include_blank => "Please select"}, { class: 'select select-primary', data: {action: "change->pair-requests-form#click"}} %>
```

Comment:
I think I removed a method out from under you in a different PR. Maybe we should just add it back? The `select` line has become a bit long and difficult to reason through. Could also kick the options to a block passed to `select`, which might look better, something like:
```erb
<%= form.select :invitee_id, nil, { :include_blank => "Please select" }, { class: "select select-primary", data: { action: "change->pair-requests-form#click"} } do %>
  <% User.excluding(current_user).each do |invitee| %>
    <%= tag.option(invitee.full_name, value: invitee.id, data: { timezone: invitee.time_zone}) %>
  <% end %>
<% end %>

Reply:
Thank you for the review Josh! I realized earlier that the method was removed so I was trying to work without it but I definitely should've checked in with you about that. I didn't know you could turn the select helper into a block to iterate over the options! It does look a lot better and I'll incorporate that ^_^